client/src/templates/Challenges/components/help-modal.tsx

```diff
@@ -31,12 +33,23 @@ const mapDispatchToProps = (dispatch: Dispatch) =>
 
 const RSA = forumLocation + '/t/19514';
 
+const generateSearchLink = (title: string, block: string) => {
+  console.table({ title, block });
```

Comment:
Is the console log left in intentional?

Reply:
Nope - I'm very good at forgetting to remove these 😅